src/ui/units/workbooks/components/Table/WorkbookEntriesTable/MainTabContent/MainTabContent.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+
+import {ChevronDown, ChevronUp, Plus} from '@gravity-ui/icons';
+import {Button, Icon} from '@gravity-ui/uikit';
+import block from 'bem-cn-lite';
+import {I18n} from 'i18n';
+import {useDispatch} from 'react-redux';
+import {CreateEntryActionType} from 'ui/units/workbooks/constants';
+import {setCreateWorkbookEntryType} from 'ui/units/workbooks/store/actions';
+
+import {ChunkGroup} from '../ChunkGroup/ChunkGroup';
+import {EmptyRow} from '../Row/Row';
+import {WorkbookEntriesTableProps} from '../types';
+import {ChunkItem} from '../useChunkedEntries';
+
+import './MainTabContent.scss';
+
+const b = block('dl-main-tab-content');
+
+const i18n = I18n.keyset('new-workbooks');
+
+interface MainTabContentProps extends WorkbookEntriesTableProps {
+    chunk: ChunkItem[];
+    actionCreateText: string;
+    title: string;
+    isErrorMessage?: boolean;
+    actionType: CreateEntryActionType;
+    isShowMoreBtn: boolean;
+    loadMoreEntries: () => void;
+    retryLoadEntries: () => void;
+}
+
+const MainTabContent: React.FC<MainTabContentProps> = ({
```

Comment:
You don't need `React.FC`. In react 17 it includes prop children. In react 18 doesn't.